docs/src/getting_started/biojuliainstallation.md

```diff
@@ -1 +1,85 @@
 # [Installing BioJulia Packages](@id biojuliainstallation)
+
+## Standard installation
+
+As with any Julia package, you can easily install any BioJulia package with the help of the [official
+package manager](https://pkgdocs.julialang.org/v1/) included by default with every Julia installation.
+All you need to know is the name of the package of interest!
+
+There are two standard methods to install a package:
+
+1. **The package manager (Pkg) module**
+
+```julia
+# You can exclude the semicolon, it is used here for suppressing output 
+using Pkg;
+# Install the package
+julia> Pkg.add("<name of package here>");
+# Use the installed package
+using <name of package here>
+```
+
+!!! note
+    One excellent trick is that any Julia code with the `julia>` prompt included at the beginning of the
+    line can be copy/pasted as is into the REPL! See the [Julia REPL](@ref julia-repl) section for more.
+
+2. **Pkg mode**
+
+```julia
+# Enter ]
+]
+# @v1.x will show your current Julia version
+(@v1.9) pkg> add <name of package here>
```

Comment:
Perhaps a note / warning about installing things in your global environment - suggest folks make project-specific environments